draft-packages/select/KaizenDraft/Select/README.mdx

```diff
@@ -36,37 +39,136 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
+*   **De-selecting:**
+    *   Single-select: where necessary, the user can de-select the selected item by clicking or tapping on a clear icon. Once removed, show placeholder text, such as “Select or search…”. Alternatively, you might require that a value is always selected so de-selecting is achieved by choosing a different option.
+    *   Multi-select: the multi-select shows selected items as tags and removes selected items from the dropdown part so that you cannot accidentally select an item twice. The user can de-select selected items using by clicking or tapping the clear icon shown in the dismissible [Tags](/components/tag).
+*   **Loading state:**
     *   We can lazy load items to be shown in the single-select on search
     *   Shows loading spinner
-*   Disabled state:
+*   **Disabled state:**
     *   See [Interaction states guidelines for disabled elements](/guidelines/interaction-states#disabled-state).
-*   Keyboard accessibility is tricky in Selects and need regular testing.
+*   **Keyboard accessibility** is tricky in Selects and needs regular testing.
+
+## Copy guidelines
+
+To be scannable and fully helpful, an individual select must have a **visible label** that describes what it is selecting.
+
+In contrast, for a **row of selects** such as a filter bar, you may use a single, shared visible label for the group, such as "Filters". You may then use visually hidden labels for each individual select to be read by screen readers and other assistive devices. In that case, use a good default for each field's pre-selected option to illustrate its use.
+
+For an individual select, here's what you need to know:
+
+### Give it a clear, short label
+
+Make sure your select has a short, descriptive label above it that describes what the user is changing by selecting an option. Keep in mind that your label shouldn’t contain instructions. Reserve those for the helper text.
+
+<DoAndDontContainer>
+<Do>
+
+- Email address
+- Full name
+
+</Do>
+
+<Dont>
+
+- Your email address
+- Full name of employee
+
+</Dont>
+</DoAndDontContainer>
+
+
+### Use minimal punctuation
+
+Don't use trailing colons and avoid phrasing your labels as questions.
+
+<DoAndDontContainer>
+<Do>
+
+- Viewed by
+- Start date
+
+</Do>
+
+<Dont>
+
+- Viewed by:
+- When did this person start?
+
+</Dont>
+</DoAndDontContainer>
+
+### For filters, be explicit
+
+To avoid ambiguity, use "by" and "to" where appropriate:
+
+<DoAndDontContainer>
+<Do>
+
+- Viewed by
+- Compared to
+
+</Do>
+
+<Dont>
+
+- Viewed
+- Comparison
+
+</Dont>
+</DoAndDontContainer>
+
+### When no option is selected
+
+In rare cases, there may be no option selected, such as in a form where "good default" makes sense, so we prompt the user to select an option. The select control will tell the user what to do instead of showing a selected option using the verb “Select”:
```

Comment:
```suggestion
In rare cases, there may be no option selected, such as in a form where “good default” makes sense, so we prompt the user to select an option. The select control will tell the user what to do instead of showing a selected option using the verb “Select”:
```